### ISSUE TITLE
[BUG Report]: An item with the same key has already been added. Key: keras_api

### ISSUE THREAD

**Issue by astronomicalphenomena** [SciSharp/TensorFlow.NET#1026]
### Description

When saving model, this error happened.
By reading Linq code, it seems like happening in TensorFlowNET.Keras.Engine.Layer.Serialize._trackable_children when concatenating model trackable with another trackable while they both have Key: keras_api

model trackable keys:
layer-0
layer-1
layer_with_weights-0
layer-2
keras_api

children keys:
non_trainable_variables
layers
variables
trainable_variables
keras_api

### Reproduction Steps

load a keras pb model saved by python and save it immediately:

var model = keras.models.load_model(@"path");
            model.summary();
            model.save(@"path");

### Known Workarounds

saveweght is fine

### Configuration and Other Information

python tensorflow version: 2.10.0-GPU
TensorFlow.Redist version:2.10.0-CPU
Tensorflow.NET: 0.100.4
Tensorflow.Keras :0.10.4
.NET: 7.0

**Comment by AsakusaRinne** [comment]
Hi, thank you for reporting us this BUG. I'll fix it soon. :)

**Comment by GadgetNutt** [comment]
I have also experienced this error setting up a Conv1D layer:
        Dim denseLayer5 = layers.Conv1D(64, kernel_size:=3, activation:="relu")
        Dim denseOutput5 = denseLayer3.Apply(denseOutput4)

**Comment by AsakusaRinne** [comment]
Here is a possible work around https://github.com/SciSharp/TensorFlow.NET/pull/976#issuecomment-1490567800. I'll fix this BUG this weekend. Recently a little busy and forgot it🤣Sorry for that

**Comment by AsakusaRinne** [comment]
#1049 partially fixed this problem. Note that tf.net support loading models with `functions/concrete functions` now but hasn't supported saving models with `functions/concrete functions`. The complete support for model saving would be added in 2 weeks. It's recommended to use model saving after loading (from model saved in python) after that.
